decontamination/benchmark_data.py

```diff
@@ -0,0 +1,135 @@
+
+"""data to filter out of the dataset"""
+import json
+import itertools
+
+from datasets import load_dataset
+
+
+MBPP_PATH = "/data/mbpp/mbpp.jsonl"
+TEST_IDS = list(range(11, 511))
+
+HUMAN_EVAL_STRINGS_OK = ['return x + y', 'return len(string)', 'return n**2']
```

Comment:
wdyt of adding `return ''.join(strings)` ?

Reply:
Sounds reasonable! If that's okay I'll add this later, so that all the christmas experiments use the same set of samples for decontamination